File: OOP/oopA2.py
class Boxer:
    def __init__(self,height,weight,wins,losses):
        self.height = height
        self.weight = weight
        self.wins = wins
        self.losses = losses

    def printStats(self):
        print(f"Height: {self.height}, Weight: {self.weight}, Wins: {self.wins}, Losses: {self.losses}")

    def printWinLossAverage(self):
        return self.wins / (self.wins + self.losses)

boxer1 = Boxer(190,200,19,0)
boxer2 = Boxer(192,205,11,1)

print("****************Boxer1******************")
boxer1.printStats()
print("**********************************")

print("*****************Boxer2*****************")
boxer2.printStats()
print("**********************************")

user_input = int( input("Which boxer would you like to bet on? Select 1 or 2") )

if(boxer1.printWinLossAverage() > boxer2.printWinLossAverage() and user_input == 1):
    print("User wins!")
else:
    print("You lose try again!")

File: ES2015+/OOPA2.js
class Boxer{
    constructor(height, weight, wins,losses){
        this.height = height
        this.weight = weight
        this.wins = wins
        this.losses = losses
    }

    printBoxerInfo(){
        console.log(`Height: ${this.height} Weight ${this.weight} Wins ${this.wins} Losses ${this.losses}`)
    }

    calculateBoxerScore(){
        return (this.height + this.weight + this.wins + this.losses) / 4
    }
}

const boxer1 = new Boxer(190, 210, 10, 0)
const boxer2 = new Boxer(188, 208, 9, 1)

console.log("***************** Boxer 1 ***********************")

boxer1.printBoxerInfo()

console.log("*************************************************")

console.log("***************** Boxer 2 ***********************")

boxer1.printBoxerInfo()

console.log("*************************************************")

const user_selection = Number( prompt("Which boxer would you like to select? Select (1 or 2)") )

let winningBoxer;

if(boxer1.calculateBoxerScore() > boxer2.calculateBoxerScore()) winningBoxer = 1
else winningBoxer= 2

if(user_selection == winningBoxer) console.log("Congrats you win!")
else console.log("Sorry you lose")